File: JavaScript/exercicio18.js
function podeVotar(){
    let mostrar = document.getElementById('mostra')
    let mostrar1 = document.getElementById('mostra1')
    let anoAtual = document.getElementById("anoAtual").value
    let anoNascimento = document.getElementById("anoNascimento").value

    let idade = Number(anoAtual - anoNascimento)

    mostrar.innerHTML = "Você tem "+idade+" anos"
    if(idade < 16){
        mostrar1.innerHTML = "Você ainda não pode votar"
    } else{
        mostrar1.innerHTML = "Você já pode votar"
    }

}

File: Java/faccat/Exercicio18.java
package faccat;

import java.util.Scanner;

public class Exercicio18 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int anoAtual, anoNascimento, idade;
		
		System.out.println("Digite seu ano de nascimento");
		anoNascimento = sc.nextInt();
		
		System.out.println("Digite o ano atual");
		anoAtual = sc.nextInt();
		
		idade = anoAtual - anoNascimento;
		System.out.println("Você tem "+idade+" anos!");
		
		if (idade >= 16) {
			System.out.println("Você pode votar!");
		} else {
			System.out.println("Você ainda não pode votar!");
		}
		
		sc.close();
	}
}
